frontend/src/pages/clerical-review/ClericalReview.css

```
@@ -79,4 +79,36 @@
 
 .listGroupItem {
     flex: 1;
+}
+
+.fixed-table-summary {
+    white-space: nowrap;
+    margin: 0;
+    border-collapse: collapse;
+    border-spacing: 0;
+    border: 1px solid black;
+    border-radius: 12px;
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+}
+
+.fixed-table-summary td,
+.fixed-table-summary th {
```

```suggestion
.fixed-table-summary td th {
```
Je sais pas si ça marche mais c'est dans la même idée que plus bas avec thead et th

---

nop c'est pas pareil que thead th un peu plus bas qui "l'un descend de l'autre". c'est pas le cas de td et th donc ça marche